### PR TITLE
redesigning project page and the markdown files of each project eg. a…

### PR DESCRIPTION
**Pull request by catsarebetter98**
Making project page look like people page
Screen Shot 2022-03-12 at 9 01 06 PM

### CODE REVIEW

File: _projects/4_digital_spatial.md [hunk 1]
```diff
@@ -1,20 +1,24 @@
 ---
-layout: page
+layout: project
 title: Tumor Infiltrating Lymphocytes
 description: Spatial Profiling of TILs with Digital Spatial Profiling
-img: assets/img/4_digitala.jpeg
+img: /ssets/img/4_digitala.jpeg
```
Comment (jlevy44) [inline _projects/4_digital_spatial.md]: typo? @catsarebetter98

**Comment by jlevy44** [comment]
@catsarebetter98 what does the projects page look like with the proposed changes?

**Comment by catsarebetter98** [comment]
@jlevy44 Project page is kept the same

File: _projects/4_digital_spatial.md (hunk 1 above)
Comment (jlevy44) [reply _projects/4_digital_spatial.md]: @catsarebetter98

**Comment by jlevy44** [comment]
There is a typo, see above. I can fix

**Pull request merged**